WebToPay.php

```diff
@@ -1843,10 +1867,14 @@ public function getCallbackValidator() {
             if (!isset($this->configuration['projectId'])) {
                 throw new WebToPay_Exception_Configuration('You have to provide project ID');
             }
+            if (!isset($this->configuration['password'])) {
```

Comment:
I see one small problem with that - password is not required for WebToPay_Sign_SS2SignChecker and for clients everything works without password in such case. But now you require it for all signers (+ your new feature) so current feature may suddenly break for them.  Could you please make password string|null and check for it only when decoding data with your new feature?